src/main/index.ts

```diff
@@ -17,6 +17,12 @@ import { fileExists } from "./fileExists";
 import { setupListeners } from "./listeners";
 import { menu } from "./menu";
 
+// On macOS, we need to force Electron to use Metal if possible. Without this flag, OpenGL will be used...
+// in software rendering mode. This has a notable impact on animations on Catalina and Big Sur.
+if (isMac) {
+  app.commandLine.appendSwitch("enable-features", "Metal");
```

Comment:
Is this fine for High Sierra users?

Reply:
I believe it should be, since it's ultimately telling Chromium to use Metal if it can - and I trust Chrome to get that part right across platforms.

That said, I do not have a High Sierra test machine to try this on... and frankly that's unsupported by Apple. I understand wanting to match what mainline Dolphin supports with regards to Ishiiruka, but I'm not interested in dealing with High Sierra here - so someone else can do that legwork if it matters enough.

Reply:
i finally found the sections of code that figure out if Metal should be used and it does handle checking if metal is supported falling back if it won't work.